src/main/resources/application-jpa.yml

```
@@ -2,6 +2,9 @@ spring:
   config:
     activate:
       on-profile: jpa
+  sql:
+    init:
+      mode : always
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)